test/functionalTest/cases/1715_batch_update/batch_update_many.test

```diff
@@ -0,0 +1,523 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+V2 batch update MANY
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+
+#
+# 01. Create E1/T1/A1=1, E2/T2/A2=2, E3/T3/A3=3, E4/T4/A4=4, using POST /v2/op/update with APPEND
+# 02. GET /v2/entities to see all four entities
+# 03. Add attributes A1=1 for E2,E3 and E4 using POST /v2/op/update with APPEND
+# 04. GET /v2/entities to see that all four entities have the attribute A1
+# 05. 'Change' the A1 attributes using POST /v2/op/update with UPDATE, all different values the the A1s
+# 06. GET /v2/entities to see that is worked
+# 07. Replace attributes for A6 in all four entities using POST /v2/update?options=keyValues with REPLACE
+# 08. GET /v2/entities to see that all four entities have only one attribute: A6
+# 09. DELETE the attribute A6 in E1, E2 and E4
+# 10. GET /v2/entities E3/A6 and the other entities without attributes
+# 11. DELETE entities E1 and E3
+# 12. GET /v2/entities, to see E2 and E4 only
+#
+
+
+echo "01. Create E1/T1/A1=1, E2/T2/A2=2, E3/T3/A3=3, E4/T4/A4=4, using POST /v2/op/update with APPEND"
+echo "==============================================================================================="
+payload='{
+  "actionType": "APPEND",
+  "entities": [
+    {
+      "id": "E1",
+      "type": "T1",
+      "A1": 1
+    },
+    {
+      "id": "E2",
+      "type": "T2",
+      "A2": 2
+    },
+    {
+      "id": "E3",
+      "type": "T3",
+      "A3": 3
+    },
+    {
+      "id": "E4",
+      "type": "T4",
+      "A4": 4
+    }
+  ]
+}'
+orionCurl --url '/v2/op/update?options=keyValues' --payload "$payload" --json
```

Comment:
Looking to NGSIv2 .apib, options=keyValues is not included in the /op/update operation. Please, add it (hint: copy-paste from an operation that already has it in the .apib).